48 modifications based on aw feedback (#50)
siamakn/temp-nomad-onboarding-workshop: docs/Module2/M2_x_explore_use_cases.md

> # Exploring Data through Use Cases (Apps) in NOMAD

Reviewer: ```suggestion
# Exploring Data Through Use Cases (Apps) in NOMAD

```

- **Heterogeneous Catalysis:** Focuses on data related to catalytic processes, including catalyst design, reaction mechanisms, and efficiency.
- **Batteries:** Encompasses data on battery materials, electrochemical performance, and simulations relevant to energy storage technologies.

## How to Explore Data Using the **USE CASES** Page

The **USE CASES** page in the EXPLORE menu is designed to facilitate exploring data in that specific field. Once you choose a use case, a predefined dashboard containing various important widgets which offer key information and visualizations to explore and analyze the data (that comply with that use case schema) is presented. Each research topic may have details and intricacies that the predefined dashboard does not address. However, NOMAD offers high flexibility, allowing you to edit the dashboard to meet your needs or add your own visualizations.

## Advanced Tips for Using the **USE CASES** Page

> To make the most out of the use cases feature, consider the following tips:

Reviewer: The customize your search point may not be very accurate. To the best of my knowledge, these customization cannot be saved at the moment. Please confirm with a developer, and update as needed.

Reviewer: Sn-based perovskite solar cells

> **Which HTL materials improve efficiency in Sn-based solar cells with C60 as the ETL?**

Reviewer: ```suggestion
**Which hole transporting layer (HTL) materials improve efficiency in Sn-based solar cells with C60 as the ETL?**

```

Reviewer: Also add (ETL) in the above section when you mention electron tranporting layer

> 1. Start by navigating to the **EXPLORE** menu in the NOMAD GUI. From here, select **Solar Cells** under use cases.

Reviewer: ```suggestion
1. Start by navigating to the **EXPLORE** menu in the NOMAD landing page. From here, select **Solar Cells** under use cases.

```
